api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -148,4 +149,7 @@ message TcpProxy {
   // The maximum number of unsuccessful connection attempts that will be made before
   // giving up. If the parameter is not specified, 1 connection attempt will be made.
   google.protobuf.UInt32Value max_connect_attempts = 7 [(validate.rules).uint32 = {gte: 1}];
+
+  // Optional configuration for TCP proxy hash policy.
+  repeated type.HashPolicy hash_policy = 11;
```

Comment:
I thought we agreed to put a max repeated constraint here of 1 for now to simplify the implementation?

Reply:
Yeah, I didn't resolve the last comment yet. I will add the constraint in the next commit.

Reply:
Update the constraint to allow at most one hash policy.